L/LinearFold/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, BinaryBuilderBase, Pkg
```

Comment:
It doesn't hurt, but I think `BinaryBuilderBase` isn't actually needed?

Reply:
No it isn't needed, i removed it. Thanks for the review!